test/integration-test-main.sh

```diff
@@ -21,6 +21,7 @@
 
 set -o errexit
 set -o pipefail
+set -o errtrace
```

Comment:
Thanks, we forgot to set this.

Reply:
Sorry I removed this lets add it in a separate PR if necessary.